README.md

```diff
@@ -156,6 +157,18 @@ As standard `SVProgressHUD` offers two preconfigured styles:
 
 If you want to use custom colors use `setForegroundColor` and `setBackgroundColor:`. These implicity set the HUD's style to `SVProgressHUDStyleCustom`.
 
+## iPhone 7 Haptics
```

Comment:
Just call it "Haptic Feedback" as Apple does (https://developer.apple.com/ios/human-interface-guidelines/interaction/feedback/). This will make it more future proof.